Update WPK documentation: adapt new names and paths (#7246)
wazuh/wazuh-documentation: source/user-manual/agent/agent-management/remote-upgrading/agent-upgrade-module.rst

    |                 +-------------------+---------------+----------+------------------------------+----------------------------------------------------------------------------------+
    |                 | **use_http**      | true, false   | no       | false                        | Whether retrieve the WPK file over http or https                                 |
    |                 +-------------------+---------------+----------+------------------------------+----------------------------------------------------------------------------------+
    |                 | **force_upgrade** | true, false   | no       | false                        | Forces the agents to upgrade, ignoring package type and version validations      |
    |                 +-------------------+---------------+----------+------------------------------+----------------------------------------------------------------------------------+

>     |                 | **package_type**  | string        | no       | Manager-inferred             | Specify the type of package to use to upgrade the agents (rpm, deb),             |

Reviewer: ```suggestion
    |                 | **package_type**  | string        | no       | Manager-inferred             | Specifies the package type for upgrading agents. For example rpm and deb.        |
```

>     |                 |                   |               |          |                              | especially useful for upgrading unrecognized system platforms                    |

Reviewer: ```suggestion
    |                 |                   |               |          |                              | Especially useful for upgrading unrecognized system platforms                    |
```
